app/views/judgment/judgmentFaq.scala.html

```diff
@@ -0,0 +1,258 @@
+@(isLoggedIn: Boolean, name: String)(implicit messages: Messages, request: RequestHeader)
+
+@main("judgmentFAQ", isLoggedIn = isLoggedIn, name = name, isJudgmentUser = true) {
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">Frequently Asked Questions</h1>
+            <h2 class="govuk-heading-m" id="getting-started">Getting started</h2>
+            <h3 class="govuk-heading-s">What is TDR?</h3>
+            <p class="govuk-body">
+                Transfer Digital Records (TDR) is The National Archives’ secure digital transfer service that enables Public Record Bodies to upload, prepare and then transfer selected digital public records to The National Archives (TNA) for permanent archiving and publication.
+            </p>
+            <p class="govuk-body">
+                For judgments, TDR is used to transfer the approved judgment to TNA for publication via their Case Law service.
+            </p>
+
+            <h3 class="govuk-heading-s">How does transferring a judgment record work?</h3>
```

Comment:
Bit random but is it worth putting ids on the heading so we can reference a particular bit of the page.
`<h3 id=how-does-it-work...` and then we can do `...gov.uk/faq#how-does-it-work`
Don't know if it's worth the effort so feel free to ignore me.